rfc/rfc021-vp_token-grant-type.md

```diff
@@ -117,15 +114,14 @@ The second paragraph describes the requirements that apply to JWT encoded VP's.
 
 1. The assertion MUST be a valid JWT according to §6.3.1 of the Verifiable Credentials Data Model 1.1 [VC-DATA-MODEL].
 2. The `vp` field of the JWT MUST be valid according to §4.10 the Verifiable Credentials Data Model 1.1 [VC-DATA-MODEL].
-3. The `nonce` of the JWT MUST be a string that is unique for each access token request.
-4. The `iss` field MUST be a Decentralized Identifier [DID].
-5. The `kid` header MUST be a DID URL and MUST resolve to a verificationMethod in the DID Document. The DID part of the DID URL MUST match the `iss` field.
-6. The `sub` field MUST match the `credentialSubject.id` field from all the Verifiable Credentials that are used to request the access token.
-7. The `aud` field MUST match the DID of the Authorization Server.
-8. The `iat` field MUST be present and contain a valid timestamp. It MUST be before the current time.
-9. The `exp` field MUST be present and contain a valid timestamp. It MUST be after the current time.
-10. The difference between the `exp` and `iat` fields MUST be equal or less than 5 seconds.
-11. The `jti` field MUST be present and contain a string that is unique for each access token request.
+3. The `iss` field MUST be a Decentralized Identifier [DID].
+4. The `kid` header MUST be a DID URL and MUST resolve to a verificationMethod in the DID Document. The DID part of the DID URL MUST match the `iss` field.
```

Comment:
I saw some DID URLs (I think verificationMethod IDs) that just contained a fragment. We might have to support that in (near) future